new module prescribing opioids for chronic pain and treatment of OUD.

yanheras opened the pull request:
Add a new module prescribing opioids for chronic pain and treatment of Opioids Use Disorder. As the prescribing opioids for chronic pain module applies to year 2014 and after modified the Opioid Addiction module from guard >=1990 to guard >=1990 and < 2014. The following is the description of the new module:
"This module models the prescribing of opioids for chronic pain and treatment of opioid use disorder (OUD) for patients age >= 18. It is based on the Centers for Disease Control (CDC) Guideline for Prescribing Opioids for Chronic Pain. This CDC guideline provides recommendations for the prescribing of opioid pain medication by primary care clinicians for chronic pain (i.e., pain conditions that typically last >3 months or past the time of normal tissue healing) in outpatient settings outside of active cancer treatment, palliative care, and end-of-life care. The applicable year of this module is set to 2014 and after. This module does not address the transition from acute pain to chronic pain. The Treatment of OUD component of this module is modeled based on the American Society of Addiction Medicine (ASAM) National Practice Guideline for the Use of Medications in the Treatment of Addiction Involving Opioid Use." Please see attached detailed documentation about this new module.  
[20200923_Module_Companion_Guide_Prescribing_Opioids_for_Chronic_Pain_and_Treatment_of_OUD.docx](https://github.com/synthetichealth/synthea/files/5308087/20200923_Module_Companion_Guide_Prescribing_Opioids_for_Chronic_Pain_and_Treatment_of_OUD.docx)

jawalonoski commented:
@yanheras Thank you for the pull request. We are a little behind reviewing and merging pull requests these days. Thank you for your patience!

jawalonoski commented:
Apologies for the delay. Clinician review was quick, but the delay was me and my schedule. So, I apologize for that.

Some findings are attached below, but I can summarize here:

- Please provide the condition as "reason" for prescribed medications and procedures.
  - For example, see the [Examplitis Module](https://synthetichealth.github.io/module-builder/#example_module), where the Examplitol MedicationOrder and the Examplotomy Procedure both provide the "reason" as the Examplitis ConditionOnset.
- There is a question regarding the percentage of patients with chronic pain who received pharmacologic treatment.
- There appears to be an unexplained gap with the absence of naltrexone.

[PrescribingOpioidReview_2020-10-14.docx](https://github.com/synthetichealth/synthea/files/5430107/PrescribingOpioidReview_2020-10-14.docx)

yanheras commented:
Thanks for the clinical review feedback! 

Couple questions: 
1) Since there is no mention of technical review feedback, does this mean the module has passed any technical checks and a new pull request would be approved for merging once we address the clinical review feedback?  
2) Regarding the comment related to adding the Condition as the "reason": this module has different chronic pain condition(s) pathways that lead to the same set of medicationOrder and procedure states (it was modeled this way to help avoid proliferations in number of states having to create). Synthea does not seem to support dynamically setting a condition as "reason" at the design time, it would not be correct if we have to choose one of the conditions as "reason". Since "reason" is an optional attribute, is it okay to not include Condition as the "reason" for this module?

yanheras commented:
Updated the module to address the clinical review feedback
1. Added OUD as reason for medication orders and procedures related to OUD.  Did not add reason for medication orders and procedures related to chronic pain because multiple chronic pain conditions could lead to the same medication order and procedures depending on the pathway. 
2. Added nalrexone to the module
3. Adjusted the prevalence rate of addiction per Vowles et al. 2015

jawalonoski commented:
Thank you for the pull request!
